rules/escape-case.js

```diff
@@ -17,6 +23,48 @@ const fix = value => {
 	return value;
 };
 
+/**
+ * Find the [start, end] position of the lowercase escape sequence in regular expression literal ASTNode
+ * @param {string} value a string representation of a literal ASTNode
+ * @return {number[]|null} [start, end] pair if found, null if not found
+ */
+const findLowercaseEscape = value => {
+	const ast = parseRegExpLiteral(value);
+	let escapeNodePosition = null;
+	visitRegExpAST(ast, {
+		onCharacterLeave(node) {
+			if (escapeNodePosition) {
+				return;
+			}
+
+			const matches = node.raw.match(escapePatternWithLowercase);
+
+			if (matches && matches[2].slice(1).match(hasLowercaseCharacter)) {
+				escapeNodePosition = [node.start, node.end];
```

Comment:
You cannot use `.start` and `.end`. See: https://github.com/sindresorhus/eslint-plugin-unicorn/issues/272

Reply:
Here is different to `fixer.replaceTextRange([node.start, node.end])` as the `node` here is actually the `ASTNode` from `regexpp`, while other `node` is the AST node from eslint parsers. 

As `start` and `end` is [official properties](https://github.com/mysticatea/regexpp/blob/v2.0.1/src/ast.ts#L66) of a `ASTNode` in `regexpp`, I think we can leave it as-is, but I can add a comment here as a reminder.

Reply:
Ah ok. I missed that. It's ok to leave it. My mistake.